test/parallel_api/ranges/merge_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/pstl_test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+
+#if _ONEDPL_USE_RANGES
+#include _PSTL_TEST_HEADER(ranges)
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ONEDPL_USE_RANGES
+    const int max_n = 10;
+    int data1[max_n] = { 0,  1, 2, 3, 4, 5, 6, 6, 6, 6};
```

Comment:
Here, we have test-case with only non-empty arrays with fixed size. Do we need to add an additional test-cases with the different sizes (including empty) or non-range APIs cover it?

Reply:
The non-range APIs (to be more precise  - hetero patterns) cover it.